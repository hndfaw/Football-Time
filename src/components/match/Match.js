import React, {Component} from 'react';
import './match.css';

  export class Match extends Component {
  
  render() {
    const {match} = this.props
    // const date = match.event_date.split("").slice(0, 10)
  
  return (
    <section className="match-container">
      <div className="match-second-container">
      <p className="team-name home-team-name">{match.homeTeam.team_name}</p>
      <div className="logos-result-container">
        <div className="home-team-logo-container">
          <img src={match.homeTeam.logo} alt="home team logo" className="small-logo" />
        </div >
        <h4 className="results">{match.goalsHomeTeam}<span> - </span>{match.goalsAwayTeam}</h4>
        <div className="away-team-logo-container">
          <img src={match.awayTeam.logo} alt="away team logo" className="small-logo" />
        </div>
      </div>
      <p className="team-name away-team-name">{match.awayTeam.team_name}</p>
      </div>
      {(match.statusShort !== 'MF' && match.statusShort !== 'NS' && match.statusShort !== 'TBD') ? <div className="match-status">
        <p className="status-content">{match.elapsed}'</p>
      </div> : <p className="date">{match.event_date}</p>}
    </section>
  )
  }
}


export default Match