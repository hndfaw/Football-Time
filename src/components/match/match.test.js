import React from 'react';
import { shallow } from 'enzyme';
import {Match} from './Match';



describe('Match', () => {
  
    it.skip('should match the match snapshot', () => {
      // let mockMatch = {
      //   event_date: "2019-08-31T16:30:00+00:00",
      //   league_id: 754,
      //   team_name: 'team'
      // }
      let wrapper = shallow(<Match match={mockMatch} />)
      expect(wrapper).toMatchSnapShot()
    })
})