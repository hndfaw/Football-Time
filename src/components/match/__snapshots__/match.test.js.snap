// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match should match the match snapshot 1`] = `
<section
  className="match-container"
>
  <div
    className="match-second-container"
  >
    <p
      className="team-name home-team-name"
    />
    <div
      className="logos-result-container"
    >
      <div
        className="home-team-logo-container"
      >
        <img
          alt="home team logo"
          className="small-logo"
        />
      </div>
      <h4
        className="results"
      >
        <span>
           - 
        </span>
      </h4>
      <div
        className="away-team-logo-container"
      >
        <img
          alt="away team logo"
          className="small-logo"
        />
      </div>
    </div>
    <p
      className="team-name away-team-name"
    />
  </div>
  <p
    className="match-status"
  />
</section>
`;
